README.md

```diff
@@ -89,3 +89,18 @@ var mocha = new Mocha({
     }
 });
 ```
+
+You can also configure the `testsuites.name` attribute by setting `reporterOptions.fileSuiteTitle` and the root suite's `name` attribute by setting `reporterOptions.rootSuiteTitle`.
```

Comment:
I guess you wanted to say
* "rootSuiteName" instead of "rootSuiteTitle"
* "testsuitesName" instead of "fileSuiteTitle"

Btw. "...SuiteTitle" would be more consistent with the already existing option "suiteTitleSeparedBy".